lms/templates/components/card/card.underscore

```diff
@@ -2,40 +2,40 @@
 <div class="wrapper-card-core">
     <div class="card-core">
         <% if (pennant) { %>
-            <small class="card-type"><%- pennant %></small>
+            <small class="card-type"><%= HtmlUtils.escape(pennant) %></small>
         <% } %>
         <h3 class="card-title"
             <% if (!_.isUndefined(srInfo)) { %>
                 aria-describedby="<%= srInfo.id %>"
             <% } %>
-            ><%- title %>
+            ><%= HtmlUtils.escape(title) %>
         </h3>
-        <p class="card-description"><%- description %></p>
+        <p class="card-description"><%= HtmlUtils.escape(description) %></p>
     </div>
 </div>
 <div class="wrapper-card-meta has-actions">
     <div class="card-meta">
     </div>
     <div class="card-actions">
-        <a class="action <%= action_class %>" href="<%= action_url %>"><%= action_content %></a>
+        <a class="action <%= action_class %>" href="<%= action_url %>"><%= HtmlUtils.escape(action_content) %></a>
```

Comment:
Again, all `<%=` in the file should be `<%-` (e.g. action_class and action_url) unless it is combined with one of your helpers.